detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/timers/JavaTimersReflected.kt

```diff
@@ -0,0 +1,26 @@
+package com.wix.detox.reactnative.idlingresources.timers
+
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.modules.core.JavaTimerManager
+import org.joor.Reflect
+import kotlin.reflect.full.declaredFunctions
+import kotlin.reflect.jvm.isAccessible
+
+object JavaTimersReflected {
+
+    fun hasActiveTimers(reactContext: ReactContext): Boolean {
+        val timersManager = getTimersManager(reactContext)
+        val hasActiveTimersInRangeInstanceClass = timersManager::class
+        val method =
+            hasActiveTimersInRangeInstanceClass.declaredFunctions.first { it.name.contains("hasActiveTimersInRange") }
+        method.isAccessible = true
+        val hasActiveTimers: Boolean = method.call(timersManager, BUSY_WINDOW_THRESHOLD) as? Boolean ?: false
+        return hasActiveTimers
+    }
+
+    private fun getTimersManager(reactContext: ReactContext): JavaTimerManager {
+        val reactHost = Reflect.on(reactContext).field("mReactHost").get<Any>()
+        val reactInstance = Reflect.on(reactHost).field("mReactInstance").get<Any>()
+        return Reflect.on(reactInstance).field("mJavaTimerManager").get() as JavaTimerManager
```

Comment:
Hey @gosha212 👋 

Saw two things here:

1- they converted this file to kotlin (first released in react-native 0.80) and it changed from `mJavaTimerManager` to the non-Hungarian notation `javaTimerManager`:

https://github.com/facebook/react-native/blame/0e6b94f4ab62a5cfa9cb770615e255fca31221f9/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactInstance.kt#L101

Maybe 

```kotlin

        try {
          return Reflect.on(reactInstance).field("mJavaTimerManager").get() as JavaTimerManager
        } catch (nsfe: NoSuchMethodException) {
          return Reflect.on(reactInstance).field("javaTimerManager").get() as JavaTimerManager
        }
```